Example/UINotifications-Example/ViewController.swift

```diff
@@ -34,11 +34,11 @@ enum NotificationStyle: UINotificationStyle {
             return .white
         }
     }
-    
+
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Example/UINotifications-Example/ViewController.swift;
  Line: 37;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)